packages/app/src/App.tsx

```diff
@@ -58,7 +66,13 @@ const routes = (
       path="/cost-insights/labeling-jobs"
       element={<CostInsightsLabelDataflowInstructionsPage />}
     />
-    <Route path="/docs" element={<TechdocsPage />} />
+    <Route path="/docs" element={<TechDocsIndexPage />}>
+      + <DefaultTechDocsHome />+{' '}
```

Comment:
This line looks a bit weird with the `+` 🤔

Reply:
Oh nice catch! Sloppy copy/paste from changelog diffs. Surprisingly the app booted up and seemed to work fine in manual testing 😮

Reply:
Yeah that's surprising!